### PR TITLE
feat(instance.stop): update modal text content

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-407 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/instances/instance/stop/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
   "pci_projects_project_instances_instance_stop_title": "Arrêt de votre instance",
   "pci_projects_project_instances_instance_stop_submit_label": "Confirmer",
   "pci_projects_project_instances_instance_stop_cancel_label": "Annuler",
-  "pci_projects_project_instances_instance_stop_content": "Vous allez arrêter votre instance {{ instance }}. Les ressources dédiées à votre instance Public Cloud sont toujours réservées (adresse IP incluse). Vous pouvez redémarrer votre instance à tout moment. Dans l'intervalle, vous êtes toujours facturé au même prix pour votre instance.",
+  "pci_projects_project_instances_instance_stop_content_1": "Vous allez arrêter votre instance {{ instance }}. Les ressources dédiées sont toujours réservées (adresse IP incluse). Vous pouvez redémarrer votre instance à tout moment.",
+  "pci_projects_project_instances_instance_stop_content_2": "Attention : Cette action entrainent toujours une facturation de l’instance tant que celle-ci n’est pas <strong>supprimée</strong>.",
```
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/instances/instance/stop/translations/Messages_fr_FR.json:6]:
```suggestion
  "pci_projects_project_instances_instance_stop_content_2": "Attention : Cette action entraine toujours une facturation de l'instance tant que celle-ci n'est pas <strong>supprimée</strong>.",
```

File: packages/manager/modules/pci/src/projects/project/instances/instances.controller.js [hunk 2]
```diff
@@ -51,6 +51,14 @@ export default class CloudProjectComputeInfrastructureListCtrl {
     );
   }
 
+  getBadgeTooltipText(status) {
+    return status === 'STOPPED' || status === 'SUSPENDED'
+      ? this.$translate.instant(
+          `pci_projects_project_instances_status_${status.toUpperCase()}_TOOLTIP`,
```
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/instances/instances.controller.js:57]:
```suggestion
          `pci_projects_project_instances_status_${status}_TOOLTIP`,
```
As you compare status to strings in upper case, it means status is already in upper case format

**Review by darsene — CHANGES_REQUESTED** [review]

**Review by JacquesLarique — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instance/stop/constants.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export const STOP_INSTANCE_INFO_LINK =
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/instances/instance/stop/constants.js:1]: **nitpick(blocking):** We cannot add the link only in French. This has to be added in all languages.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instance/stop/constants.js (hunk 3 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/instances/instance/stop/constants.js:1]: I'll check this with the PO.

Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/instances/instance/stop/constants.js]: @anooparveti 
The links to the other subsidiaries is addded in the last commit.

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/instances/instance/stop/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -2,7 +2,9 @@
   "pci_projects_project_instances_instance_stop_title": "Arrêt de votre instance",
   "pci_projects_project_instances_instance_stop_submit_label": "Confirmer",
   "pci_projects_project_instances_instance_stop_cancel_label": "Annuler",
-  "pci_projects_project_instances_instance_stop_content": "Vous allez arrêter votre instance {{ instance }}. Les ressources dédiées à votre instance Public Cloud sont toujours réservées (adresse IP incluse). Vous pouvez redémarrer votre instance à tout moment. Dans l'intervalle, vous êtes toujours facturé au même prix pour votre instance.",
+  "pci_projects_project_instances_instance_stop_content_1": "Vous allez arrêter votre instance {{ instance }}. Les ressources dédiées sont toujours réservées (adresse IP incluse). Vous pouvez redémarrer votre instance à tout moment.",
+  "pci_projects_project_instances_instance_stop_content_2": "Attention : Cette action entrainent toujours une facturation de l'instance tant que celle-ci n'est pas <strong>supprimée</strong>.",
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/instances/instance/stop/translations/Messages_fr_FR.json:6]:
```suggestion
  "pci_projects_project_instances_instance_stop_content_2": "Attention : Cette action entraine toujours une facturation de l'instance tant que celle-ci n'est pas <strong>supprimée</strong>.",
```

**Review by anooparveti — APPROVED** [review]

**Pull request closed without merging**